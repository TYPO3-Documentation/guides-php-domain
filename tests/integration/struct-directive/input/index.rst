=================
PHP static method
=================

..  php:namespace:: App\Accounting

..  php:struct:: Employee

    ..  php:property:: firstName
        :type: string

    ..  php:property:: lastName
        :type: string

    ..  php:property:: salary
        :type: int

    ..  php:property:: fullTime
        :type: bool
