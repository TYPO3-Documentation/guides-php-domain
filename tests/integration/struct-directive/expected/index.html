<!-- content start -->
    <div class="section" id="php-static-method">
    <h1>PHP static method</h1>

            <dl class="php struct">
    <dt class="sig sig-object php"
        id="app-accounting-employee">
        <em class="property"><span class="pre">struct</span> </em>
            <span class="sig-prename descclassname"><span class="pre">\App\Accounting\</span></span>
<span class="sig-name descname"><span class="pre">Employee</span></span>

    </dt>
    <dd>
        <dl class="php property">
    <dt class="sig sig-object php" id="firstname">
        <span class="pre">string</span> <span class="sig-name descname"><span class="pre">firstName</span></span>

    </dt>
    <dd></dd>
</dl>
<dl class="php property">
    <dt class="sig sig-object php" id="lastname">
        <span class="pre">string</span> <span class="sig-name descname"><span class="pre">lastName</span></span>

    </dt>
    <dd></dd>
</dl>
<dl class="php property">
    <dt class="sig sig-object php" id="salary">
        <span class="pre">int</span> <span class="sig-name descname"><span class="pre">salary</span></span>

    </dt>
    <dd></dd>
</dl>
<dl class="php property">
    <dt class="sig sig-object php" id="fulltime">
        <span class="pre">bool</span> <span class="sig-name descname"><span class="pre">fullTime</span></span>

    </dt>
    <dd></dd>
</dl>

    </dd>
</dl>

    </div>

<!-- content end -->
